### PR TITLE
Update Sessions page with new filters and columns.

### PR DESCRIPTION
**Pull request by stbof**
- DOC-3532
- DOC-5697
- DOC-5529
- DOC-5446

### CODE REVIEW

**Review by maryliag — APPROVED** [review]
:lgtm:

> Reviewed 2 of 2 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/15080#-:-NBnmouK291edPvQLh5D:bnfp4nl)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @stbof)

File: _includes/v22.2/ui/sessions.md [hunk 1]
```diff
@@ -6,24 +6,27 @@ To filter the sessions, click the **Filters** field.
...
-To filter by [application]({{ link_prefix }}connection-parameters.html#additional-connection-parameters), select **App** and choose one or more applications.
+- To filter by [application]({{ link_prefix }}connection-parameters.html#additional-connection-parameters), select **App** and choose one or more applications.
 
-- Queries from the SQL shell are displayed under the `$ cockroach` app.
-- If you haven't set `application_name` in a client connection string, it appears as `unset`.
+    - Queries from the SQL shell are displayed under the `$ cockroach` app.
+    - If you haven't set `application_name` in a client connection string, it appears as `unset`.
 
-To filter by session duration, specify the session time and unit.
+- To filter by username or session status, select **Username** or **Session Status** and choose one or more checkboxes.
```
Comment (kevin-v-ngo) [inline _includes/v22.2/ui/sessions.md:14]: Nit: User name 

image

File: _includes/v22.2/ui/sessions.md [hunk 2]
```diff
@@ -6,24 +6,27 @@ To filter the sessions, click the **Filters** field.
 
 <img src="{{ 'images/v22.2/ui-session-filter.png' | relative_url }}" alt="Session filter" />
 
-To filter by [application]({{ link_prefix }}connection-parameters.html#additional-connection-parameters), select **App** and choose one or more applications.
+- To filter by [application]({{ link_prefix }}connection-parameters.html#additional-connection-parameters), select **App** and choose one or more applications.
 
-- Queries from the SQL shell are displayed under the `$ cockroach` app.
-- If you haven't set `application_name` in a client connection string, it appears as `unset`.
+    - Queries from the SQL shell are displayed under the `$ cockroach` app.
+    - If you haven't set `application_name` in a client connection string, it appears as `unset`.
```
Comment (taroface) [inline _includes/v22.2/ui/sessions.md:12]:
```suggestion
    - If you have not set `application_name` in a client connection string, it appears as `unset`.
```

File: _includes/v22.2/ui/sessions.md (hunk 1 above)
Comment (taroface) [inline _includes/v22.2/ui/sessions.md:14]: Nit: It feels a little weird to "choose" a checkbox, but I grant that "check" may be redundant. Maybe "fill" or "click"?

```suggestion
- To filter by username or session status, select **Username** or **Session Status** and check one or more checkboxes.
```

File: _includes/v22.2/ui/sessions.md [hunk 3]
```diff
@@ -6,24 +6,27 @@ To filter the sessions, click the **Filters** field.
...
-The following are displayed for each session:
+The following properties are displayed for each session:
 
 Column | Description
 --------- | -----------
 Session Start Time (UTC) | The timestamp at which the session started.
 Session Duration | The amount of time the session has been open.
-Status  | The status of the session: Active or Idle. A session is Active if it has an open explicit or implicit transaction (individual SQL statement) with a statement that is actively running or waiting to acquire a lock. A session is Idle if it is not executing a statement.
+Session Active Duration | The amount of time transactions were executing while the session was open.
+Status  | The status of the session: Active, Closed, or Idle. A session is Active if it has an open explicit or implicit transaction (individual SQL statement) with a statement that is actively running or waiting to acquire a lock. A session is Closed if it has closed the connection to CockroachDB. A session is Idle if it is not executing a statement.
```
Comment (taroface) [inline _includes/v22.2/ui/sessions.md:26]: If "Active", "Closed" and "Idle" are the actual status names, I think they should be in backticks.

**Review by taroface — APPROVED** [review]
LGTM; just some nits.

File: _includes/v22.2/ui/sessions.md (hunk 1 above)
Comment (stbof) [reply _includes/v22.2/ui/sessions.md:14]: @kevin-v-ngo The checkbox in the filter is labeled **Username**. See the screenshot ui-session-filter.png in this PR.

File: _includes/v22.2/ui/sessions.md (hunk 3 above)
Comment (stbof) [inline _includes/v22.2/ui/sessions.md:26]:
```suggestion
Status  | The status of the session: `Active`, `Closed`, or `Idle`. A session is `Active` if it has an open explicit or implicit transaction (individual SQL statement) with a statement that is actively running or waiting to acquire a lock. A session is `Closed` if it has closed the connection to CockroachDB. A session is `Idle` if it is not executing a statement.
```

File: _includes/v22.2/ui/sessions.md (hunk 1 above)
Comment (stbof) [reply _includes/v22.2/ui/sessions.md]: @kevin-v-ngo  I filed this: https://github.com/cockroachdb/cockroach/issues/87960

**Pull request merged**